### PR TITLE
Key quote escaping

### LINKED ISSUE #22: Quotes in keys are not escaped
I am using English translation as keys. When a translation has a quote in it, the quote is not escaped - and therefore, the format of .strings file is invalid, and project can't build.

For `|en:Say "hello"|de:Sag "hallo"|`, I expect to get `"Say \"hello\"" = "Sag \"hallo\"";`
And what I get now is `"Say "hello"" = "Sag \"hallo\"";`.

### PR DESCRIPTION
**Pull request by ghost**
Resolves https://github.com/AckeeCZ/ACKLocalization/issues/22

#### Checklist
- [x] Added tests (if applicable)

### CODE REVIEW

**Comment by ghost** [comment]
This fixes https://github.com/AckeeCZ/ACKLocalization/issues/22

**Review by olejnjak — APPROVED** [review]
Hi, thanks for the PR 🙂

For me it looks just fine 🙂

**Comment by LukasHromadnik** [comment]
Hi! Thanks for the PR. Looks good to me. 👍

**Review by LukasHromadnik — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,15 @@
 
 ## master
 
+## 1.1.3
+
+### Fixed
+- Fix creating invalid keys, if contain quote marks ([#22](https://github.com/AckeeCZ/ACKLocalization/pull/22))
```
Comment (fortmarek) [inline CHANGELOG.md]: You're just missing `kudos @mateuszjablonski` here 🙂

**Review by fortmarek — APPROVED** [review]
Minor nitpick concerning changelog, otherwise LGTM 🎉

File: CHANGELOG.md (hunk 1 above)
Comment (ghost) [reply CHANGELOG.md]: updated 👍

**Review by janmisar — APPROVED** [review]

**Pull request merged**